### DISCUSSION TITLE
医療機関・検査機関の検索をしたい

### DISCUSSION THREAD

**Discussion by keiji** [cocoa-mhlw/cocoa#880] [Ideas]
「COCOA」アプリの接触通知が届きました - ケータイWatch
https://k-tai.watch.impress.co.jp/docs/column/minna/1379515.html

> PCR検査や抗原検査ができる医療機関が検索できる画面に進めるのですが、接続されたWebページはパソコン向けのブラウザに最適化されたもので、iPhoneで見ると、「Excel」アプリをインストールしていないと医療機関のリストが正しく表示されなかったり、タップ操作がしづらかったり、地図を拡大するとデータの読み出しにものすごく時間を要したりと、なにかとストレスがたまる仕様でした。

現在のCOCOAは、接触通知を受けた後「検査等の相談先を探す」に遷移した後はWebコンテンツになりますが、そこでは詳細な検索には対応していません。
また、掲載しているのはコールセンターや保健所などの公的機関の番号が主で、医療機関は各自治体のウェブサイトに遷移することになります（ケータイWatchの記事は自治体のサイトの話と思われます）。

各自治体のサイトや配信しているデータについて、直接どうこうと言うことはできないと認識しています。

その上で、医療機関・検査機関の検索をCOCOAから直接できるようにすれば、利用者の体験を大きく改善できるのではないかと考えています。

たとえば、東京都であれば「診療・検査医療機関の一覧」がオープンデータになっています。

診療・検査医療機関の一覧
https://catalog.data.metro.tokyo.lg.jp/dataset/t000010d0000000095

いきなり全国は無理でも、まずはオープンデータがある東京都だけでも、医療機関を検索できるようにしてはどうかと考えています。

まずはベータ版で配信して評判を見て、用途が見通せるようならプロダクト版に載せつつ、他の地域にも広げる（※）ことも検討したいと考えています。また、接触通知の有無にかかわらず、COCOAを起動すれば医療機関・検査機関を検索できるようになれば、COCOA以外からの利用もできるかもしれません。

※ どうやって広げるかはノープランです。各地方自治体に協力をお願いするのは厳しいかもしれません。誰でも登録可能なウェブサイトを作ってボランタリーで登録、事実確認後に公開みたいなフローを回せればいいのですが。

**Comment by b-wind** [comment]
現状表示されるコンテンツはこれですかね。
https://covid19radarjpnprod.z11.web.core.windows.net/ja/chatbot2.html

ここから自分に取って都合の良い相談先を見つけるのも難しいと思いますので、手助けになるようなサイトは有ると良いかなとは思います。

**Reply by keiji** [reply]
ぼくの場合、どうしてもアプリに組み込むことを考えてしまいますが、独立したサイトとして運営して、それをCOCOAから見る形でもいいですね。
